controlplane/rosa/api/v1beta2/rosacontrolplane_types.go

```diff
@@ -76,6 +77,169 @@ type RosaControlPlaneSpec struct { //nolint: maligned
 	// - ocmApiUrl: Optional, defaults to 'https://api.openshift.com'
 	// +optional
 	CredentialsSecretRef *corev1.LocalObjectReference `json:"credentialsSecretRef,omitempty"`
+
+	// PodCIDR
+	// +kubebuilder:validation:Format=cidr
+	// +optional
+	PodCIDR string `json:"PodCIDR,omitempty"`
+
+	// ServiceCIDR
+	// +kubebuilder:validation:Format=cidr
+	// +optional
+	ServiceCIDR string `json:"serviceCIDR,omitempty"`
+
+	// HostPrefix
+	// +optional
+	HostPrefix int `json:"hostPrefix,omitempty"`
+
+	// The compute machine type to use, for example `r5.xlarge`.
+	// +optional
+	ComputeMachineType RosaMachineType `json:"computeMachineType,omitempty"`
```

Comment:
this should just be a string indicating the instance type name.

Reply:
This is a bit confusing , so is there a reference for the machineType names ?